carbonj.service/Dockerfile

```diff
@@ -9,6 +9,10 @@ FROM quay.io/centos/centos:stream8
 ARG DEPENDENCY=build/docker/dependency
 ARG VERSION=latest
 
+RUN cd /etc/yum.repos.d/
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
```

Comment:
While you are touching this file, can you please consolidate all RUN into a single command? Each RUN produce a layer on the image which is insane.

Reply:
This actually doesn't work because the base is centos8